src/schemas/action.schema.json

```diff
@@ -232,6 +235,23 @@
                 "text",
                 "ability"
             ]
+        },
+        "writeToEditorAction": {
```

Comment:
This is good but you may want to make the descriptions more detailed and technical because this is what will be given to system prompts to get the proper LLM output. So extra detail here goes a long way. Stuff like the specific WordPress API that will be used to stream the blocks or if there's a specific version of the block formats